interface/ceed-operator.c

```diff
@@ -82,6 +84,11 @@ int CeedOperatorCreate(Ceed ceed, CeedQFunction qf, CeedQFunction dqf,
   @param op         Ceedoperator on which to provide the field
   @param fieldname  Name of the field (to be matched with the name used by CeedQFunction)
   @param r          CeedElemRestriction
+  @param lmode      CeedTransposeMode which specifies the ordering of the
```

Comment:
I don't really understand this explanation. Is it for row major and column major?

Reply:
This is actually part of PR #170. In this PR we can support two orderings of vectors with multiple fields - either all of the data for a particular dof is contiguous (field is the innermost dimension of the vector ordering) or all of the data of a particular field is contiguous (field is the outermost dimension of the vector ordering)